api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/components/OperationResult.java

```diff
@@ -11,6 +11,7 @@ public class OperationResult {
     private static final OperationResult OK = new OperationResult(HttpServletResponse.SC_OK);
 	private static final OperationResult INTERNAL_SERVER_ERROR = new OperationResult(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
 	private static final OperationResult METHOD_NOT_ALLOWED = new OperationResult(HttpServletResponse.SC_METHOD_NOT_ALLOWED);
+	private static final OperationResult NOT_AUTHORIZED = new OperationResult(HttpServletResponse.SC_UNAUTHORIZED);
```

Comment:
can we align static field name with HttpServerlResponse.SC name (as in lines above and below)

Reply:
Sure